test/blackbox-tests/test-cases/pkg/substitute.t

```diff
@@ -75,32 +77,28 @@ Now with variables set
   > '%%{doc}%%' is '%{doc}%'
   > '%%{stublibs}%%' is '%{stublibs}%'
   > '%%{man}%%' is '%{man}%'
-  > Finally, '%%{custom}%%' is '%{custom}%'
   > EOF
   $ cat >dune.lock/test.pkg <<EOF
   > (source (copy $PWD/test-source))
   > (build
   >  (progn
-  >   (withenv
-  >    ((= custom defined-here))
-  >    (substitute defined.ml.in defined.ml))
+  >   (substitute defined.ml.in defined.ml)
```

Comment:
Where is this assignment tested now?

Reply:
That assignment just didn't make sense so I get rid of it. One is not allowed to substitute environment variables like that.